src/Class Specs/grid/main.css

```diff
@@ -0,0 +1,30 @@
+.MainData {
+  display: grid;
+  height: 100%;
+  width: auto;
+  grid-column: 1fr 1fr 1fr;
+  grid-template-columns: 1fr 1fr 1fr;
+  grid-template-rows: 50% 50%;
+}
```

Comment:
This looks nice, however instead of using template columns, their is template area in case you want to change around or work around that.

Reply:
That is true, however, that might not always work with multiple elements being dynamically placed with fetch apis